compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.cpp

```diff
@@ -856,35 +856,6 @@ LogicalResult MMAAttr::materializeOperandConcreteShape(
 // DataTiledMMA Attributes
 //===----------------------------------------------------------------------===//
 
```

Comment:
Not in the diff here, but I think you can also get rid of `DataTiledMMAAttr::getUnrollingFactor` now, since the unrolling factors are carried by the attribute (or you can update the implementation to just return the carried unrolling factors).

Reply:
Thanks for the catch!